crates/e2e/src/tests.rs

```diff
@@ -0,0 +1,15 @@
+#[test]
```

Comment:
What we really want to do is fully test the `#[ink_e2e::test(ws_url = "…")]` macro here

Reply:
It is not possible to use E2E macros outside the contract's project:
```
ERROR: Cannot infer the root project id
[2022-11-10T20:19:57Z ERROR ink_e2e_macro::codegen] `cargo-contract` stderr: 
error: custom attribute panicked
 --> crates/e2e/tests/sample.rs:5:5
  |
5 |     #[ink_e2e::test(ws_url = "ws://0.0.0.0:9944")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: contract build for Cargo.toml failed

error: could not compile `ink_e2e` due to previous error
```